src/lib.ts

```diff
@@ -506,3 +507,18 @@ export function getPlayerFromIdOrName(idOrName: number | string): Player {
     ? { name: idOrName, id: parseInt(getPlayerId(idOrName)) }
     : { name: getPlayerName(idOrName), id: idOrName };
 }
+
+/**
+ * Tries to get an effect using the default method
+ * @param ef effect to try to get
+ * @param turns turns to aim for; default of 1
+ */
+export function ensureEffect(ef: Effect, turns = 1): void {
+  if (haveEffect(ef) < turns) {
+    if (!cliExecute(ef.default) || haveEffect(ef) === 0) {
+      throw `Failed to get effect ${ef.name}.`;
```

Comment:
I understand why throwing here is necessary. Let's make an `EnsureError` or something, and throw that. And make sure its exported by libram too.